src/gui/SearchDialog.cpp

```diff
@@ -667,6 +672,73 @@ void SearchDialog::setSimpleStyle()
 	ui->coordinateSystemComboBox->setVisible(false);
 }
 
+void SearchDialog::setCenterOfScreenCoordinates()
+{
+	StelCore *core = StelApp::getInstance().getCore();
+	const auto projector = core->getProjection(StelCore::FrameJ2000, StelCore::RefractionMode::RefractionOff);
+	Vec2i centerScreen(projector->getViewportPosX() + projector->getViewportWidth() / 2,
+			   projector->getViewportPosY() + projector->getViewportHeight() / 2);
```

Comment:
Bad alignment in editors with 4-space tabs. We seem to have agreed to use the "indent with tabs, align with spaces" rule, see [here](https://github.com/Stellarium/stellarium/pull/3284#issuecomment-1626946051).